ProcessMaker/Http/Controllers/Api/TemplateController.php

```diff
@@ -98,6 +101,9 @@ public function updateTemplate(string $type, Request $request)
     public function updateTemplateConfigs(string $type, Request $request)
     {
         $request->validate(Template::rules($request->id, $this->types[$type][4]));
+        //Call event to log Template Config changes
+        $changes=$request->only(['id', 'name', 'description','process_category_id']);
+        TemplateUpdated::dispatch($changes,null,'Configure Template');
```

Comment:
use a const instead of a hardcoded value ''Configure Template'' like:
`TemplateUpdated::CONFIGURE_TEMPLATE`